cli/cli/kurtosis_package/kurtosis_package.go

```diff
@@ -0,0 +1,104 @@
+package kurtosis_package
+
+import (
+	"fmt"
+	"github.com/go-yaml/yaml"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/shared_utils"
+	"github.com/kurtosis-tech/stacktrace"
+	"github.com/sirupsen/logrus"
+	"os"
+	"path"
+)
+
+const (
+	kurtosisPackageFilePermissions os.FileMode = 0644
+	kurtosisYmlFilename                        = "kurtosis.yml"
+	mainStarFilename                           = "main.star"
+	mainStarFileContentStr                     = `def run(plan):
+    # TODO
```

Comment:
can we do ` plan.print("hello world!") instead

1. they get something functional
2. they can see how `plan` works

Reply:
good suggestion. I'll do it